decidim-meetings/lib/decidim/meetings/admin_engine.rb

```diff
@@ -31,8 +31,8 @@ class AdminEngine < ::Rails::Engine
             end
           end
           resources :agenda, except: [:index, :destroy]
-          resources :attachment_collections
-          resources :attachments
+          resources :attachment_collections, except: [:show]
+          resources :attachments, except: [:show]
```

Comment:
We would need to add this same change in the other modules:

```shell
$ grep -ri "resources :attachment" decidim*/lib/
decidim-accountability/lib/decidim/accountability/admin_engine.rb:          resources :attachment_collections
decidim-accountability/lib/decidim/accountability/admin_engine.rb:          resources :attachments
decidim-assemblies/lib/decidim/assemblies/admin_engine.rb:          resources :attachment_collections, controller: "assembly_attachment_collections"
decidim-assemblies/lib/decidim/assemblies/admin_engine.rb:          resources :attachments, controller: "assembly_attachments"
decidim-blogs/lib/decidim/blogs/admin_engine.rb:          resources :attachment_collections
decidim-blogs/lib/decidim/blogs/admin_engine.rb:          resources :attachments
decidim-budgets/lib/decidim/budgets/admin_engine.rb:          resources :attachment_collections
decidim-budgets/lib/decidim/budgets/admin_engine.rb:          resources :attachments
decidim-conferences/lib/decidim/conferences/admin_engine.rb:          resources :attachment_collections, controller: "conference_attachment_collections"
decidim-conferences/lib/decidim/conferences/admin_engine.rb:          resources :attachments, controller: "conference_attachments"
decidim-elections/lib/decidim/votings/admin_engine.rb:          resources :attachments, controller: "voting_attachments"
decidim-elections/lib/decidim/votings/admin_engine.rb:          resources :attachment_collections, controller: "voting_attachment_collections"
decidim-initiatives/lib/decidim/initiatives/admin_engine.rb:          resources :attachments, controller: "initiative_attachments"
decidim-meetings/lib/decidim/meetings/admin_engine.rb:          resources :attachment_collections
decidim-meetings/lib/decidim/meetings/admin_engine.rb:          resources :attachments
decidim-participatory_processes/lib/decidim/participatory_processes/admin_engine.rb:          resources :attachment_collections, controller: "participatory_process_attachment_collections"
decidim-participatory_processes/lib/decidim/participatory_processes/admin_engine.rb:          resources :attachments, controller: "participatory_process_attachments"
```

Reply:
Thanks @andreslucena, I was already on the case! I couldn't convert the PR back into a draft! Thanks for the listing the modules still needing these changes.